packages/appstore/src/components/add-more-wallets/wallet-add-card.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Text, WalletCard } from '@deriv/components';
+import { useCurrencyConfig, useDisplayAvailableWallets } from '@deriv/hooks';
+import { getAddWalletDetails } from 'Constants/add-wallet-card-content';
+
```

Comment:
```suggestion

const wallet_description_mapper = {
    USD: localize(
        'Deposit and withdraw US dollars into your accounts regulated by MFSA using credit or debit cards and e-wallets.'
    ),
    EUR: localize(
        'Deposit and withdraw euros into your accounts regulated by MFSA using credit or debit cards and e-wallets.'
    ),
    ...
};
```
@aizad-deriv You can create a mapper here in the component itself.